### ISSUE TITLE
Need for Pascal VOC config file

### ISSUE THREAD

**Issue by Yuxin-Du-Lab** [Megvii-BaseDetection/DenseTeacher#10]
The released code just include coco config. I wonder if you can give the VOC related code, or just the configuration file is also OK.

I tried to use the configuration about VOC given in the paper,setting VOC 2007 train as labeled data & VOC 2012 train as unlabeled data.But I got an unstable training process.

Set trainer as follow:

│ TRAINER               │ {'DISTILL': {'GAMMA': 2.0,                                                          
│                       │              'RATIO': 0.01,                                                         
│                       │              'SUPPRESS': 'linear',                                                  
│                       │              'SUP_WEIGHT': 1,                                                       
│                       │              'UNSUP_WEIGHT': 1,                                                     
│                       │              **'WEIGHTS': {'DELTAS': 1.0, 'LOGITS': 2.0, 'QUALITY': 1.0}},**
│                       │  'EMA': {'DECAY_FACTOR': 0.9996, 'START_STEPS': 3000, 'UPDATE_STEPS': 1},           
│                       │  'NAME': 'SemiRunner',                                                              
│                       │  'SSL': {'BURN_IN_STEPS': 5000}}   

Is that right?

Could you help me, please

**Comment by ZRandomize** [comment]
Yes, the weight is right, but we used EMA at 10000 step and BURN_IN_STEP=20000 since VOC is a full dataset. An early burn-in stage can cause unstable training due to the low-quality pseudo-labels

**Comment by Yuxin-Du-Lab** [comment]
Thank you very much! I'll try it again with new hyper-para.

**Comment by Yuxin-Du-Lab** [comment]
I still got some problem in training using the new super-para.
Part of tensorboard info:
![total loss](https://user-images.githubusercontent.com/60123629/187102869-dbdeadbc-dd9e-4899-9cdb-858c74a9df03.PNG)
![AP50](https://user-images.githubusercontent.com/60123629/187102873-bfd36880-8b04-4c5c-81ef-4d92eeddec11.PNG)
The min value point in total loss figure is about 20000 steps, namely getting a divergence training process after 'BURN_IN'.
The detailed log file is following:
[voc_log.txt](https://github.com/Megvii-BaseDetection/DenseTeacher/files/9440841/voc_log.txt)

What else is wrong? I need your help, please.

**Comment by ZRandomize** [comment]
seems you are using voc2007 val as eval set, but we use trainval for training and voc2007 test as evaluation set.
beside, the best performance is obtained at 60K iter since the voc set is small.
Our training curve is like this (AP50):
![image](https://user-images.githubusercontent.com/49480962/187109592-2ef523c3-f1c5-4244-a67a-87461158bfcb.png)

**Comment by Yuxin-Du-Lab** [comment]
Thanks for your detailed reply! I just tried the new division of dataset and got a better score: 76.01(AP50).
![ap50](https://user-images.githubusercontent.com/60123629/188122535-82abf80e-d98e-4bb0-bb04-fdd20c89839f.png)

But you know, it's still lower than the score presented in paper(79.89).
I wonder if there any config that I ignored.

[config.py file](https://github.com/Megvii-BaseDetection/DenseTeacher/files/9477241/config.file.txt)
[detailed log header file](https://github.com/Megvii-BaseDetection/DenseTeacher/files/9477242/detailed.log.header.txt)
[voc dataset class replacement file](https://github.com/Megvii-BaseDetection/DenseTeacher/files/9477243/dataset.replacement.txt)

Thanks for your help, again!

**Comment by ZRandomize** [comment]
ah, seems there's only one more step, we used Scale Jittering as in Unbiased Teacher like [here](https://github.com/Megvii-BaseDetection/DenseTeacher/blob/ca5c29d04674fdb68c389839a2f8d9ec637ad4c9/coco-full/config.py#L91)
You can also choose Large Scale Jittering as in Soft Teacher like in [here](https://github.com/Megvii-BaseDetection/DenseTeacher/blob/ca5c29d04674fdb68c389839a2f8d9ec637ad4c9/coco-full-LSJ/config.py#L91)
